lighthouse-core/test/runner-test.js

```diff
@@ -260,7 +260,7 @@ describe('Runner', () => {
 
     return Runner.run({}, {url, config}).then(results => {
       const audits = results.audits;
-      assert.equal(audits['critical-request-chains'].displayValue, 9);
+      assert.equal(audits['critical-request-chains'].displayValue, 5);
```

Comment:
this is the good kind of test change, right? :D 

(i.e. we were wrong before and now we're right)

Reply:
yes, you can see the diff here.
https://www.diffchecker.com/ec6WBOxh

As you can see we removed 3 iframes from the critical requests:
- https://www.redditmedia.com/ads/display/300x250-companion
- https://www.redditmedia.com/gtm/jail?cb=8CqR7FcToP
- https://www.redditmedia.com/ads/display/300x250

so in total we removed 5 leaf nodes but we only remove 4 chains as `https://www.redditstatic.com/reddit-init.en.belOwpfqWtw.js`is still a root node.
- https://www.redditstatic.com/ad-dependencies.5O7sMdAReBw.js
- https://www.redditstatic.com/companion.KW1mOM2pJek.js
- https://www.redditstatic.com/gtm.aX_QHhLRPyo.js
- https://www.redditstatic.com/ad-dependencies.5O7sMdAReBw.js
- https://www.redditstatic.com/display.PGSStRgaC6w.js